pandas/_libs/tslibs/timestamps.pyx

```diff
@@ -538,12 +538,19 @@ cdef class _Timestamp(ABCTimestamp):
         return bool(ccalendar.is_leapyear(self.year))
 
     @property
-    def dayofweek(self) -> int:
+    def day_of_week(self) -> int:
         """
         Return day of the week.
         """
         return self.weekday()
 
+    @property
```

Comment:
You don't need to define a second property, set `dayofweek` to be an alias of `day_of_week`